hudi-common/src/main/java/org/apache/hudi/common/table/view/TableFileSystemView.java

```diff
@@ -157,6 +157,8 @@ interface SliceView extends SliceViewWithLatestSlice {
 
   /**
    * Stream all the file groups for a given partition.
+   * <p>
+   * Note: This method will return all file groups in a partition, i.e. uncommitted filegroups will be returned
```

Comment:
Why does this add the note? IMO, the description of this interface doesn't mentition that the return value doesn't include uncommitted filegroups.

Reply:
Sorry, it was a WIP thing while i was debugging this error.